.github/workflows/ci.yml

```diff
@@ -420,14 +420,13 @@ jobs:
           components: miri
       - uses: Swatinem/rust-cache@v2
       - name: miri
-        # Many of tests in tokio/tests and doctests use #[tokio::test] or
-        # #[tokio::main] that calls epoll_create1 that Miri does not support.
         run: |
-          cargo miri test --features full --lib --no-fail-fast
+          rustup toolchain install nightly nightly-2024-09-19 --component miri --profile minimal --no-self-update
+          rustup override set nightly
+          cargo miri test --features full --lib --tests --no-fail-fast
```

Comment:
I am trying to keep the change minimal, so I don't really want to change global nightly version (i.e. ``env.rust_nightly``), but I am not sure if this will work.